packages/wallet/contract/src/utils.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+
+import { assert, details as X, quote as q } from '@agoric/assert';
+import { deeplyFulfilledObject, objectMap } from '@agoric/internal';
+import { E } from '@endo/far';
+
+/**
+ *
+ * @param {AmountKeywordRecord} amountKeywordRecord
+ * @param {(brand: Brand) => ERef<Purse>} purseForBrand
+ * @returns {PaymentPKeywordRecord}
+ */
+export const paymentsFromPurses = (amountKeywordRecord, purseForBrand) => {
```

Comment:
Nit: Rename to `withdrawPaymentsFromPurses` so it's clear that this is performing an action

Reply:
good call!